pubsub/subscriptions/subscription_test.go

```diff
@@ -123,23 +123,32 @@ func setup(t *testing.T) *pubsub.Client {
 
 func TestCreate(t *testing.T) {
 	ctx := context.Background()
-	tc := testutil.SystemTest(t)
 	client := setup(t)
-	topic, err := client.CreateTopic(ctx, topicID)
-	if err != nil {
-		t.Fatalf("CreateTopic: %v", err)
-	}
-	buf := new(bytes.Buffer)
-	if err := create(buf, tc.ProjectID, subID, topic); err != nil {
```

Comment:
It looks like this is the actual test of the sample, and it's not included in your new version of the test...